lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -211,8 +333,59 @@
 .active-filter-item {
     display: flex;
     justify-content: space-between;
+    margin: 0.5rem 0.2rem;
+
+    button {
+        margin-right: -5px;
+    }
+}
+
+.no-properties {
+    padding: 1rem 0;
 }
 
 .pagination-buttons {
     display: flex;
+    padding: 2rem;
+    justify-content: flex-end;
+    height: fit-content;
+}
+
+// for tablet displays
+@media screen and (min-width: 650px) {
+    .left-panel-mobile {
+        width: 60vw;
+    }
+}
+
+// for sizing between mobile and desktop
+@media screen and (min-width: 899px) and (max-width: 1250px) {
+    .search-page > div:first-of-type {
+        margin: 0 auto;
+    }
+
+    .heading-wrapper > label {
+        padding-left: 0;
+        margin: 0;
+    }
+
+    .search-main {
+        gap: 1rem;
+    }
+
+    .object-type-nav {
+        display: flex;
+        flex-direction: column;
+        height: fit-content;
+    }
+
+    .sort-dropdown {
+        align-self: flex-end;
+        margin: 0.5rem 0;
+
+        > div {
+            background-color: $color-bg-white;
+            margin-top: 0.4rem;
+        }
+    }
```

Comment:
I am afraid this is just adding more media queries to try to fix the problem introduced by specifying a hardcoded margin for the main gutter. It defeats the purpose of using `<OsfLayout>` and we might as well write another layout component.

Reply:
I addressed your concerns during the scrum demo and have accounted for mobile, tablet, desktop, and the in-between browser client window re-sizings. OsfLayout is actually pretty helpful and it was one line of code to adjust it to match the mock-ups. The mobile and tablet values are actually hardcoded as well, just in another file. We can move the value or add it as a CSS variable if necessary; it was only relevant to Search so it seemed most appropriate here. The media queries would have been necessary regardless, I had not previously accounted for or tested the odd behavior present during client window re-sizings. Thank you for pointing it out and I'm glad we got it resolved.